content/en/continuous_delivery/deployments/argocd.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Monitor ArgoCD Deployments
+kind: documentation
+description: Learn how to monitor deployments from ArgoCD in Datadog Deployment Visibility.
+is_beta: true
+further_reading:
+- link: "/continuous_delivery/deployments"
+  tag: "Documentation"
+  text: "Learn about Deployment Visibility"
+- link: "/continuous_delivery/explorer"
+  tag: "Documentation"
+  text: "Learn how to query and visualize deployment executions"
+---
+
+{{< callout url="http://dtdg.co/to-be-updated" header="false" >}}
+CD Visibility for ArgoCD is in private beta. To request access, complete the form (link to be inserted).
+{{< /callout >}}
+
+## Overview
+
+[Argo CD][1] is a declarative, GitOps continuous delivery tool for Kubernetes. It follows the GitOps pattern, using Git
+repositories to define the desired application state, and it automates the deployment of applications in the specified target environments.
+
+The integration between Argo CD and Datadog CD Visibility is provided through [Argo CD Notifications][2]. In short, Argo CD notifications
+consists of two main things:
+1. [Triggers][3]: they define **when** to send a notification.
+2. [Templates][4]: they define **what** to send in a notification.
```

Comment:
```suggestion
Datadog CD Visibility integrates with Argo CD by using [Argo CD Notifications][2]. 

Argo CD notifications consists of two components:

1. [Triggers][3], which define _when_ to send a notification.
2. [Templates][4], which define _what_ to send in a notification.
```

Reply:
LGTM, I have just added "main" before components because it actually consists of other components, but those are the two main ones